packages/flutter_localizations/lib/src/l10n/cupertino_pl.arb

```diff
@@ -28,10 +28,10 @@
   "cutButtonLabel": "Wytnij",
   "copyButtonLabel": "Kopiuj",
   "pasteButtonLabel": "Wklej",
-  "selectAllButtonLabel": "Zaznacz wszystko",
+  "selectAllButtonLabel": "Wybierz wszystkie",
```

Comment:
This is incorrect. "Wybierz wszystko" refers to selecting all [countable] elements. This string is used for text selection though which should keep the existing "Zaznacz wszystko" translation. I am suspecting there could be multiple other languages with a similar translation error.

Maybe we could add a description to `selectAllButtonLabel` in _en.arb to explain what it's used for?

Reply:
I'll address this with https://github.com/flutter/flutter/issues/127871.